src/pickup.cpp

```diff
@@ -95,7 +98,16 @@ interact_results interact_with_vehicle( vehicle *veh, const tripoint &pos,
         selectmenu.addentry( CONTROL, true, 'v', _( "Control vehicle" ) );
     }
 
-    if( from_vehicle ) {
+    for( auto e : veh->get_parts( "WASHING_MACHINE" ) ) {
+        washing_machine_on = e->enabled;
```

Comment:
This won't work correctly if there are more than one washing machines installed. It will use the `enabled` status of the last one, even if you examine a different one.